app/routes/events/components/EventList.js

```diff
@@ -117,15 +118,22 @@ const filterRegDateOptions: Array<Option> = [
 ];
 
 class EventList extends Component<EventListProps, State> {
-  state = {
-    selectedOption: filterRegDateOptions[0],
-    filterEventTypesSettings: {
-      showCompanyPresentation: false,
-      showCourse: false,
-      showSocial: false,
-      showOther: false,
-    },
-  };
+  constructor(props: EventListProps) {
+    super(props);
+    const locationFilters = props.location.state.filterEventType;
```

Comment:
This can be undefined, so this won't work if you go to the url without the `filterEventType` :upside_down_face:.

Something like this should work f.ex: (although kinda ugly)
```suggestion
    const locationFilters = props.location.state.filterEventType || [];
```